projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/meta_dag.py

```diff
@@ -34,12 +35,19 @@ def __init__(self, team_name: str):
                 )
             ),
         )
+        self._max_concurrent_starting_jobs = int(
```

Comment:
```suggestion
        self._max_concurrent_running_jobs = int(
```

Isn't ita better name ?

Reply:
You are right, thank you. Done.